release-content/0.15/migration-guides/14082_AssetReader_read_now_returns_an_opaque_type.md

```diff
@@ -0,0 +1,38 @@
+The trait method `bevy_asset::io::AssetReader::read` (and `read_meta`) now return an opaque type instead of a boxed trait object. Implementors of these methods should change the type signatures appropriately
```

Comment:
```suggestion
The trait method `bevy_asset::io::AssetReader::read` (and `read_meta`) now return an opaque type instead of a boxed trait object. Implementors of these methods should change the type signatures appropriately:
```